packages/docs/src/pages/components.js

```diff
@@ -24,12 +33,13 @@ const ComponentGroup = ({ groupName, components }) => {
       key={slugify(component.node.frontmatter.title)}
       title={component.node.frontmatter.title}
       html={component.node.html}
+      id={`${groupName.replace(' ', '-')}-${component.node.frontmatter.title.replace(' ', '-')}`}
```

Comment:
These `.replace` should use regex with the global flag to match all of the instances (replace all of the spaces): `.replace(/\s/g, '-')`. Right now, it will only replace the first space, resulting in things like `#App-Icons-Extra-Large Icons` (the space between "Large" and "Icons")

Reply:
Why not use `slugify`?